frontend/src/components/Search/SimpleSearch/SimpleSearch.logic.ts

```diff
@@ -0,0 +1,49 @@
+import type { QueryParams } from "~/stores/searchParams";
+import { DocumentKind } from "~/types";
+
+export function getUrl(category?: string) {
+  if (category?.startsWith(DocumentKind.CaseLaw)) {
+    return `/v1/case-law`;
+  } else if (category?.startsWith(DocumentKind.Norm)) {
+    return `/v1/legislation`;
+  } else {
+    return `/v1/document`;
+  }
+}
+
+/**
+ * Converts the params used in the pinia store to URL params as expected by
+ * the API.
+ */
+export function convertParams(
+  params: Omit<QueryParams, "dateSearchMode"> & { temporalCoverage?: string },
+) {
```

Comment:
Wouldn't we benefit here from a defined return type to the function? then we can know exactly how the returned json looks like, also the const result Record would have better typing rather than string types.

Reply:
True, that would make things clearer indeed, and prevent bugs in case we want to refactor this.